creator-node/src/fileManager.js

```diff
@@ -33,17 +29,14 @@ async function saveFileFromBufferToDisk(req, buffer) {
     throw new Error('User must be authenticated to save a file')
   }
 
-  // Retrieve multihash
-  const multihash = await ipfsAdd.generateNonImageMultihash(
-    buffer,
-    req.logContext
-  )
+  // Retrieve CID
+  const cid = await fileHasher.generateNonImageMultihash(buffer, req.logContext)
 
-  // Write file to disk by multihash for future retrieval
-  const dstPath = DiskManager.computeFilePath(multihash)
+  // Write file to disk by cid for future retrieval
+  const dstPath = DiskManager.computeFilePath(cid)
   await fs.writeFile(dstPath, buffer)
 
-  return { multihash, dstPath }
+  return { cid, dstPath }
```

Comment:
noted. just double-checked again to make sure that both usages of this function refer to `cid` instead of `multihash` -- there's a test for this as well